app/lib/pre_assembly/bundle.rb

```diff
@@ -105,8 +105,6 @@ def digital_objects
         )
         params[:object_files] = discover_object_files(params[:stageable_items])
         DigitalObject.new(params).tap do |dobj|
-          p manifest_rows
-          p "helwlekjrwelkj"
```

Comment:
aha - yes, you caught it.  Perhaps this commit could be merged into first one?  Don't really care that much right now, but review has been a bit sub-optimal.